napari/layers/_tests/test_layer_actions.py

```diff
@@ -37,13 +38,25 @@ def _dummy():
 )
 def test_projections(mode):
     ll = LayerList()
-    ll.append(Image(np.random.rand(8, 8, 8)))
+    ll.append(Image(np.random.rand(6, 7, 8)))
     assert len(ll) == 1
     assert ll[-1].data.ndim == 3
-    _project(ll, mode=mode)
+    dims = Dims(ndim=ll[-1].data.ndim, order=(0, 1, 2))
+    _project(ll, dims=dims, mode=mode)
```

Comment:
I know this wasn't here before and the default is 0, but it helps to specify the parameter value explicitly to make this test easier to reason about.

```suggestion
    _project(ll, dims=dims, axis=0, mode=mode)
```